lib/wrappers/python/contourwall.py

```diff
@@ -111,34 +166,99 @@ def drop(self) -> None:
         self._drop(ctypes.byref(self._cw_core))
 
 def hsv_to_rgb(hue: int, saturation: float, value: float) -> tuple[int, int, int]:
```

Comment:
This function shoul be replaced with this. Make sure you improve the naming and such


```python
def hsv_to_rgb(h, s, v):
    h /= 360
    s /= 100
    v /= 100
    
    if s == 0.0: return v, v, v
        
    i = int(h*6.0) # XXX assume int() truncates!
    f = (h*6.0) - i
    i = i%6
    
    p = round((v*(1.0 - s)) * 255)
    q = round((v*(1.0 - s*f)) * 255)
    t = round((v*(1.0 - s*(1.0-f))) * 255)
    v = round(v*255)
    
    if i == 0:
        return v, t, p
    if i == 1:
        return q, v, p
    if i == 2:
        return p, v, t
    if i == 3:
        return p, q, v
    if i == 4:
        return t, p, v
    if i == 5:
        return v, p, q
```

Reply:
make sure you also mention in a comment that we stole this implementation from Colorsys:

https://github.com/python/cpython/blob/3.12/Lib/colorsys.py